etc/.golangci.yaml

```diff
@@ -17,43 +18,50 @@ linters:
     - contextcheck
     - cyclop
     - deadcode
+    - depguard
     - exhaustivestruct
     - exhaustruct
     - forcetypeassert
     - funlen
-    - gocognit
-    - godox
-    - goerr113
     - gochecknoglobals
     - gochecknoinits
+    - gocognit
+    - goconst
     - gocyclo
+    - godox
+    - goerr113
     - gofmt
     - goimports
     - golint
     - gomnd
     - ifshort
     - importas
+    - inamedparam
     - interfacebloat
     - interfacer
     - ireturn
     - maintidx
-    - maligned
     - makezero
+    - maligned
     - musttag
+    - nakedret
     - nestif
     - nlreturn
-    - nosnakecase
     - nonamedreturns
+    - nosnakecase
     - nosprintfhostport
     - paralleltest
+    - perfsprint
```

Comment:
Lots of complaints about:
```
fmt.Sprintf("%d", 10)
```
It wants them to use strconv.Atoi (or Itoa) never remember which is which.